docs/Inspection-of-yum-Repositories.md

```diff
@@ -199,6 +201,23 @@ Attributes of yum repos are mapped to zypp's attributes.
 * Disadvantages:
   * Building Red Hat images not straight forward: We can find out that it is a yum repo because it doesn't have priority nor autorefresh
 
```

Comment:
Finding out if it's a yum repo by missing attributes is a fragile and error-prone solution. I don't think we should do that. I would see it as a better solution to add an explicit attribute storing the package manager which is used to manage the repository.

If this would be an optional attribute defaulting to zypp we might even get away without schema change as the new schema would be compatible with the old one.